### PR TITLE
Replace Renovate with Dependabot for dependency updates

### LINKED ISSUE #4831: Switch from Renovate to Dependabot for dependency updates
## Feature description

We've seen that Dependabot does better at updating PHP dependencies in the `composer.json`.

Also, now that we have a GitHub action which does ZIP builds for PRs, this is restricted to PRs not coming from forks, which is another benefit that Dependabot provides.

Therefore, we should decommission Renovate and switch instead to Dependabot.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation brief

* 

## QA testing instructions

* 

## Demo

* 

## Changelog entry

*

### PR DESCRIPTION
**Pull request by pierlon**
## Summary

Configures updates (in the root folder) for:
- Composer
- npm
- GitHub Actions

Fixes #4831.

## Checklist

- [x] My pull request is addressing an open issue (please create one otherwise).
- [ ] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines#tests).
- [ ] My code follows the [Engineering Guidelines](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines) (updates are often made to the guidelines, check it out periodically).

### CODE REVIEW

**Comment by pierlon** [comment]
I think it would be best to review the dependabot [docs](https://help.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates) to see if we would like to configure any other options before merging.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Specify location of manifest files for each package manager
```
Comment (swissspidy) [inline .github/dependabot.yml:1]: All the configuration options can also be added at https://app.dependabot.com/accounts/ampproject

Then there's no need for this configuration file.

We can start configuration by turning on Dependabot at https://app.dependabot.com/accounts/ampproject/add

@westonruter WDYT?

Comment (westonruter) [reply .github/dependabot.yml:1]: Yeah, I suppose so. If you think that is preferable, I'm for it.

Comment (pierlon) [reply .github/dependabot.yml:1]: I don't see any cons with that approach. If that's the case, I'll just update the PR to remove the Renovate config only.

Comment (swissspidy) [reply .github/dependabot.yml:1]: It works fine for the stories repo, and is one less config file to deal with :-)

And from what I gathered, once they integrate Dependabot into GitHub it‘ll probably be UI-only anyway.

So let‘s do it :-)

**Comment by pierlon** [comment]
PR no longer relevant.

**Pull request closed without merging**